lib/src/component_declaration/component_base.dart

```diff
@@ -15,6 +15,7 @@
 library over_react.component_declaration.component_base;
 
 import 'dart:async';
+import 'dart:collection';
```

Comment:
What was added that made this necessary?

Reply:
`MapBase` as the super class to `UiProps`